### DISCUSSION TITLE
organização do código fonte

### DISCUSSION THREAD

**Discussion by almanaque-lab** [almanaque-lab/app-ufpi-project#4] [Q&A] [answered]
Qual é a melhor maneira de organizar os arquivos de código-fonte do meu aplicativo para facilitar a manutenção e a colaboração da equipe?

**Comment by limajunio** [comment, marked as answer]
Uma boa prática é organizar os arquivos de código-fonte do seu aplicativo em uma estrutura hierárquica, agrupando-os por funcionalidade ou componente. Por exemplo, você pode ter diretórios para modelos, visualizações, controladores (no caso de frameworks MVC), recursos estáticos, templates, etc. Além disso, é útil documentar claramente a estrutura do projeto e seguir convenções de nomenclatura consistentes para facilitar a compreensão e colaboração da equipe.
